libs/anglify/src/modules/snackbar/snackbar.service.ts

```diff
@@ -0,0 +1,65 @@
+import { Overlay, OverlayConfig } from '@angular/cdk/overlay';
+import { ComponentPortal } from '@angular/cdk/portal';
+import { Injectable, InjectionToken, Injector } from '@angular/core';
+import { BehaviorSubject, Observable } from 'rxjs';
+import { SnackbarContext } from './snackbar-context.interface';
+import { SnackbarOptions } from './snackbar-options.interface';
+import { SnackbarComponent } from './snackbar.component';
+import { AnglifyIdService } from '../../services/id/id.service';
+
+export const SNACKBAR_CONTEXT = new InjectionToken<SnackbarContext>('Snackbar context');
+
+@Injectable({
+  providedIn: 'root',
+})
+export class SnackbarService {
```

Comment:
I would rename the `SnackbarService` to `Snackbar` for now, or `AnglifySnackbar` right away. 
We did the same thing in the Stepper Component (`Step`,` Stepper`). Step and Stepper should be renamed to AnglifyStep and AnglifyStepper in the future.

Reply:
Yeah I saw that the stepper is way out of line for this, but it is a service, I don't think we should do this.

We don't name our components without `Component` either. Directives are obviously an exception here, but I don't see why services shouldn't have it denoted.

Reply:
Let's just leave that for now. We can discuss this in Issue #147.